tests/test-base.bats

```diff
@@ -59,6 +59,27 @@ load test_helper
   assert_json .success true
 }
 
+@test "Harvest create form" {
+  # create a waf-collection harvest source
+  # require the organization created in previous test
```

Comment:
Can we create (and destroy) the organization as part of the test setup (and cleanup)? I'm not a fan of these "stateful" tests where one test depends on the previous one.